README.md

```diff
@@ -17,3 +28,78 @@ Fossil solves the following use cases:
 - Local Time Series Database that stores data in a local DB file
 - Collect data over a TCP connection in fire and forget mode
 - Query data over a TCP connection
+
+## How to
+
+### Install
+```shell
+go install github.com/dburkart/fossil
+```
+or
+```shell
+git clone https://github.com/dburkart/fossil.git
+cd fossil
+go run main.go
+```
+
+### Usage
+```shell
+> fossil server -h
+Database for collecting and querying metrics
+
+Usage:
+  fossil server [flags]
+
+Flags:
+  -d, --database string   Path to store database files (default "./")
+  -h, --help              help for server
+  -p, --port int          Database server port for data collection (default 8001)
+      --prom-http int     Set the port for /metrics is bound to (default 2112)
+
+Global Flags:
+  -c, --config string   Path to the fossil config file (default "./config.toml")
+  -H, --host string     Host to send the messages (default "fossil://local/default")
+      --local           Configures the logger to print readable logs (default true)
+  -v, --verbose count   -v for debug logs (-vv for trace)
+```
+
+### Config
+```toml
+[fossil]
+port = 8000
+prom-port = 2112
+
+host = "fossil://local/default"
+local = true
+verbose = 2
+
+[database]
+directory = "data/default"
+
+[database.test]
+directory = "data/mydata"
+
+[database.tester]
+```
+
+#### Root `fossil` config block
+| Option             | Default                    | Description                                            |
+| ------------------ | -------------------------- | ------------------------------------------------------ |
+| `fossil.port`      | 8000                       | Port fossil server listens on                          |
```

Comment:
Default is 8001 I think, based on the above usage?

Reply:
😅 oops